source/contributing/writing-a-tutorial.md

```diff
@@ -0,0 +1,84 @@
+# How to write a tutorial
+
+This is a guide to writing tutorials about Nix.
+
+By [tutorials](https://diataxis.fr/tutorials/) we mean *lessons* as described in the [Diátaxis framework for technical documentation](https://diataxis.fr), and recommend becoming familiar with Diátaxis before proceeding.
+Especially note [the difference between tutorials and guides](https://diataxis.fr/tutorials-how-to/).
+
+We strongly recommend the book [How Learning Works (summary)](https://www.lesswrong.com/posts/mAdMkFqWzbJRB544m/book-review-how-learning-works) as a guide for designing learning materials.
+
+## Target audience
+
+The main target audience of Nix tutorials are software developers with at least basic experience on the Linux command line.
+
+Experts answering questions immediately, personalised instructions and training, and other forms of apprenticeship are known to be the most effective support for learning Nix.
+**These tutorials are targeted at those who don't have access to any of that**, and should therefore be written to be suitable for self-directed learning.
+This is achieved by following the structure outlined here, which is primarily characterised by aiming to avoid and close all information gaps for the learner.
+
+As a byproduct, a well-written tutorial will be useful as lecture notes for use in interactive training sessions.
+Therefore, the secondary target audience are instructors teaching Nix.
+
+## Structure
+
+Each tutorial should have the following structure:
+
+### What will you learn?
+
+Describe the problem statement and learning goals.
+
+The learning goal of a tutorial is always a skill, which is distinguished by being applicable to a variety of situations that share recurrent patterns.
+
+### What do you need?
+
+State the prerequisite knowledge and skills.
+The tutorial should always be written such that the stated prerequisites are sufficient to achieve learning goals.
+
+Examples:
+
+- links to previous chapters
+- domain-specific skills or knowledge
+
```

Comment:
That should be entirely obsolete with Nix. I mean, sure, you need some sort of OS and a text editor, but since our target audience is more software developer than computer-illiterate (which is in itself a very important audience that is severely underserved!) we may leave that implied.